iot-hub/Samples/service/AutomaticDeviceManagementSample/AutomaticDeviceManagementSample.cs

```diff
@@ -25,34 +25,34 @@ public AutomaticDeviceManagementSample(RegistryManager registryManager)
         public async Task RunSampleAsync()
         {
             Console.WriteLine("Create configurations");
-            await AddDeviceConfiguration("config001").ConfigureAwait(false);
-            await AddDeviceConfiguration("config002").ConfigureAwait(false);
-            await AddDeviceConfiguration("config003").ConfigureAwait(false);
-            await AddDeviceConfiguration("config004").ConfigureAwait(false);
-            await AddDeviceConfiguration("config005").ConfigureAwait(false);
+            await AddDeviceConfigurationAsync("config001").ConfigureAwait(false);

```

Comment:
We should remove all calls to `ConfigureAwait(false)` in samples. As a rule, we don't include them because we don't know the context in which a customer will copy/paste our code into their app, and some apps (ASP.NET) can fail badly if this is used.